submissions/ravi/about.md

```diff
@@ -0,0 +1,11 @@
+# About Me
+My name is Ravi Kumar. I am a IT student at Panjab University, India. 
```

Comment:
a IT student?

Reply:
IT means information technology department